### ISSUE TITLE
Name ending with a number are not well detected

### ISSUE THREAD

**Issue by cedeber** [ArnaudBarre/eslint-plugin-react-refresh#1]
```ts
export const MyComponent2: FC = () => {
  // ..
}
```

This was detected as invalid.

**Comment by cedeber** [comment]
Thanks!

**Comment by ArnaudBarre** [comment]
Thanks for the report btw!
